pkg/cpio/newc_test.go

```diff
@@ -698,3 +699,40 @@ func FuzzReadWriteNewc(f *testing.F) {
 		}
 	})
 }
+
+// Is there an equivalent to io.Discard for readers? I thought there was.
+type EmptyReaderAt struct{}
+
+func (e EmptyReaderAt) ReadAt(p []byte, off int64) (int, error) {
+	return 0, io.EOF
+}
+
+func TestReproducibleWrite(t *testing.T) {
+	d := t.TempDir()
+	for _, n := range []string{"a", "b"} {
+		if err := ioutil.WriteFile(filepath.Join(d, n), []byte{}, 0644); err != nil {
```

Comment:
file should hava some content to check afterwards if the un-cpio-ed files have the same content or it has been lost.

Reply:
well, disregard this, I thought the tests was cpio-ing and then un-cpio-ing the files :D